install/k8s/cluster/bootstrap_centos.sh

```diff
@@ -1,20 +1,42 @@
 #!/bin/bash
 
 if [ $EUID -ne 0 ]; then
-  echo "Please run this script as root user"   
+  echo "Please run this script as root user"
   exit 1
 fi
+
 set -e
+
+swapoff -a
+setenforce 0
+systemctl stop firewalld
+systemctl disable firewalld
+/etc/init.d/network restart
+
+cp /shared/.etc_hosts /etc/hosts
```

Comment:
seems bad to have this as a hidden file?

Reply:
it was so, do you want me to change ?